scripts/release/debian/deb_install.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+#######################################################################################################################
```

Comment:
Just to clarify this warning is an in script comment. 

If users run `sudo curl -sL https://aka.ms/InstallAzureCLIDeb | bash` they won't see the warning, unless they read the script, right?

Reply:
Correct. By the time they're executing the script, the damage is already done. My thinking was that I'd rather make clear to the skeptical folks taking the time to read through this script, that the danger is not lost on us.

Reply:
@marstr The traditional way to get around that is to allow for `-sL` but also by default make the script interactive in some way, so that users can sign off on Y/N operations or otherwise inspect it before running. A lot of system administrators consider curling a bash script without reading it a serious security problem and may disallow use of the Azure CLI in their environments without that kind of support.

Reply:
10-4, I'll add some prompts to explain what we're doing.

For the record, I absolutely agree with the concern about sudo executing a bash script. My preferred solution would be to publish our product directly to Debian/Ubuntu repositories so that folks didn't have to muck with installing ours. However, for the moment I think that's a pipe-dream, so something like this will have to do.

Reply:
Have we considered adding a PPA? That's a decent middle ground for solving that kind of problem.

Reply:
Oooh, that's not a bad idea! I'll toss that around a little more after build.

Reply:
consent prompts add in 294f11e

Reply:
whats a PPA @marstr and @sptramer?

Reply:
Good question :) 
https://en.wikipedia.org/wiki/Ubuntu#Package_Archives